atom/browser/api/atom_api_session.cc

```diff
@@ -130,7 +151,27 @@ struct Converter<ClearStorageDataOptions> {
   }
 };
 
-template<>
+template <>
```

Comment:
Super minor, but looks like most other templates don't have a space before the `<>`